azure-pipelines.yml

```diff
@@ -83,7 +85,7 @@ stages:
           solution: '$(solution)'
           platform: '$(buildPlatform)'
           configuration: '$(buildConfiguration)'
-          msbuildArgs: '/p:DeployExtension=false /p:SignAssembly=true /p:AssemblyOriginatorKeyFile="$(snk.secureFilePath)" /p:Sha1=$(Build.SourceVersion) /p:BuildNumber=$(Build.BuildId) /p:WarningLevel=0'
+          msbuildArgs: '/p:DeployExtension=false /p:SignAssembly=$(isReleaseBranch) /p:AssemblyOriginatorKeyFile="$(snk.secureFilePath)" /p:Sha1=$(Build.SourceVersion) /p:BuildNumber=$(Build.BuildId) /p:WarningLevel=0'
```

Comment:
Speaking of signing, where are the JAR files actually signed?

Running this on our latest release gives `jar is unsigned.` 😒 
>jarsigner -verify sonar-csharp-plugin-8.33.0.40503.jar

Reply:
The signing should happen here: https://github.com/SonarSource/sonar-dotnet/blob/master/azure-pipelines.yml#L385 but it looks like it does not work. I've added a card in Trello for this.

Reply:
Getting back to this. We don't use the jar signer tool for our signing. Instead, the signature is detached and we use PGP signatures file which can be used to authenticate the artifacts. In this case, we have [sonar-csharp-plugin-8.33.0.40503-sources.jar.asc](https://repox.jfrog.io/artifactory/sonarsource-public-releases/org/sonarsource/dotnet/sonar-csharp-plugin/8.33.0.40503/sonar-csharp-plugin-8.33.0.40503.jar.asc).

Reply:
To sum it up, we sign now all the java artifacts. We always did. I will tackle java signing for PR branches separately.